apps/site/assets/css/_event-details.scss

```diff
@@ -16,4 +16,46 @@
   .c-paragraph {
     padding-bottom: $base-spacing * 2;
   }
+
+  .m-event__section-padding {
+    padding-bottom: $base-spacing-lg;
+  }
+  
+  .m-event__section-border {
+    border-bottom: $border;
+    &:last-child {
+      border-bottom: none;
+    }
+  }
+
+  .m-event__registration {
+    padding: $base-spacing * 2 0 $base-spacing 0;
+    
+    &-header {
+      flex: 0 1 40%;
+      margin: 0;
+      align-self: center;
+    }
+    &-container {
+      flex: 1 0 auto;
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+      align-items: center;
+    }
+
+    @include media-breakpoint-down(xs) {
+      flex-direction: column;
+      a {
+        width: 100%;
+      }
+      .m-event__registration-header {
+        align-self: unset;
+        margin-bottom: $base-spacing / 2;
+      }
+      .m-event__registration-container {
+        align-items: flex-start;
+      }
+    }
```

Comment:
It looks like each element in mobile could be displayed with display: block; to get the desired result, would that be simpler than reconfiguring flex box things?